check_license.sh

```diff
@@ -48,8 +48,37 @@ then
     cd "$1"
 fi
 
-# Known licenses must continue to match.
-shasum -a 256 -c $CHKSUM_FILE
+# Remove all newlines from the Checksum file as these are reported as formatting
+# errors by the shasum program
+TMP_CHKSUM_FILE=$(mktemp)
+trap cleanup EXIT
+cleanup()
+{
+  rm $TMP_CHKSUM_FILE
+}
+sed '/^$/d' $CHKSUM_FILE > $TMP_CHKSUM_FILE
+
+# Use the tmp-file for the rest of the script
+CHKSUM_FILE=$TMP_CHKSUM_FILE
+
+# Collect only stderr from the subcommand
+output="$(exec 3>&1; shasum --warn --algorithm 256 --check $CHKSUM_FILE >/dev/null 2>&3)"
+
+if ! grep -q 'lines are improperly formatted' 2>/dev/null < "$output"; then
+  echo >&2 "Some lines in the LIC_FILE_CHKSUM.sha256 file are misformed:"
+  (
+    IFS="
+"
+    for line in $output; do
+      if [ "$line" = "shasum: WARNING: 2 lines are improperly formatted" ]; then
+        continue
+      fi
+      linenr="$(echo $line | sed -E 's;shasum: /tmp/tmp\S+: ([0-9]+).*;\1;')"
+      awk "NR == $linenr " $CHKSUM_FILE
```

Comment:
I _think_ I understand the logic and the purpose of this. But I think is an overkill :)

TBH I don't think it is necessary to process the output so much as we risk hiding the error if the output of  `shasum` is slightly different. On failures I would just print the full `CHKSUM_FILE` as it will be quite obvious for the PR author to see what has he/she messed up.

As you have already done "the job" I don't have objections on leaving it; I just wanted to share my point of view for future reference.

Reply:
Alright. I agree. Let's keep things simple.

Reply:
KISS